swarm/storage/ldbstore.go

```diff
@@ -142,13 +142,10 @@ func NewLDBStore(params *LDBStoreParams) (s *LDBStore, err error) {
 	}
 	data, _ := s.db.Get(keyEntryCnt)
 	s.entryCnt = BytesToU64(data)
-	s.entryCnt++
```

Comment:
@zelig is there any rational for incrementing the `entryCnt` here? `entryCnt` is supposed to be the number of items in LevelDB, and here we are only initialising the `LDBStore`. Same goes for `accessCnt` and `dataIdx`.